modules/github/team.tf

```diff
@@ -20,7 +20,18 @@ locals {
   invalid_approvers = setunion(local.invalid_approvers_by_email, local.invalid_approvers_by_login)
 }
 
+locals {
+  team_id        = var.create_team ? github_team.alz[0].id : var.existing_team_name == null ? null : data.github_team.alz[0].id
+  approver_count = var.create_team ? length(local.approvers) : var.existing_team_name == null ? 0 : (data.github_team.alz[0].members)
+}
+
+data "github_team" "alz" {
```

Comment:
How will this be affected if `create_team == false` and `existing_team_name == null`?

Reply:
It sets it to 0 and does not add the team in another part of the logic..